### DISCUSSION TITLE
Disable default Tailwind Classes

### DISCUSSION THREAD

**Discussion by hejsfj** [tailwindlabs/tailwindcss#7890] [Help]
Hello,
I created a web widget with TailwindCSS and currently have the issues that there are some default classes applied by Tailwind, which mess up the original website code, where the widget gets embedded. 

I already added those lines to my tailwind config:

` prefix: process.env.TW_PREFIX,
        corePlugins: {
          preflight: false,
        },`

I attached two screenshots where you see an example of the generic classes, which are added by Tailwind and which mess up with the website where it gets embedded. 

How can I avoid that those classes are added?

<img width="1680" alt="Bildschirmfoto 2022-03-18 um 18 30 25" src="https://user-images.githubusercontent.com/47115858/159055213-f0119dd3-5315-4714-aaee-d06d4724d267.png">
<img width="1680" alt="Bildschirmfoto 2022-03-18 um 18 24 27" src="https://user-images.githubusercontent.com/47115858/159055204-f9204ab4-11c0-46d6-aa63-a93efa252807.png">

**Comment by 2van** [comment]
don't import preflight.
```
/* @tailwind base; */
/* @tailwind components; */
```

**Comment by adamwathan** [comment]
Those styles are coming from the forms plugin. I would not use that plugin if you can’t have those styles generated, or use the “class” strategy option so the styles are classes instead of base styles.
